sonatype/src/main/scala/org/typelevel/sbt/TypelevelSonatypePlugin.scala

```diff
@@ -79,6 +82,24 @@ object TypelevelSonatypePlugin extends AutoPlugin {
       }
   }
 
+  private lazy val sonatypeApiUrl = Def.setting {
+    if (publishArtifact.value)
+      CrossVersion(
+        crossVersion.value,
+        scalaVersion.value,
+        scalaBinaryVersion.value
+      ).map { cross =>
+        val host = sonatypeCredentialHost.value
+        val repo = if (isSnapshot.value) "snapshots" else "releases"
+        val org = organization.value.replace('.', '/')
+        val mod = cross(moduleName.value)
+        val ver = version.value
+        url(
+          s"https://$host/service/local/repositories/$repo/archive/$org/$mod/$ver/$mod-$ver-javadoc.jar/!/index.html")
```

Comment:
Interestingly, the `index.html` at the end of that doesn't cause problems. I'm inclined to keep it.